app/lib/package/backend.dart

```diff
@@ -1169,9 +1177,13 @@ class PackageBackend {
   }
 
   Future<api.SuccessMessage> removeUploader(
-      String packageName, String uploaderEmail) async {
+    String packageName,
+    String uploaderEmail, {
+    AuthSource? authSource,
+  }) async {
+    authSource ??= AuthSource.client;
     uploaderEmail = uploaderEmail.toLowerCase();
-    final user = await requireAuthenticatedUser();
+    final user = await requireAuthenticatedUser(source: authSource);
```

Comment:
Can the client trigger this anymore?

Reply:
Old clients still can, we are not actively blocking on it yet.